go/protocol/keybase1/extras.go

```diff
@@ -1786,3 +1786,17 @@ func LockIDFromBytes(data []byte) LockID {
 	sum := sha512.Sum512(data)
 	return LockID(binary.LittleEndian.Uint64(sum[:8]))
 }
+
+// MDPriority is the type for the priority field of a metadata put. mdserver
+// prioritizes MD writes with higher priority when multiple happen at the same
+// time, for the same TLF.
+const (
+	// MDPriorityDefault is the priority of zero. It's implicitly used by all
+	// old clients, and has lowest priority.
+	MDPriorityDefault MDPriority = 0
+	// MDPriorityNormal is the priority used for normal KBFS metadata writes.
+	MDPriorityNormal = 8
```

Comment:
Why is normal different from the default?  You want to start prioritizing new clients over old ones?

Reply:
Yeah that's one thing it does, despite not being very useful. The other thing is that we can't use placeholders at all if the priority is 0, so I thought to make it work the same way we should use non-zero priorities from now on.